src/trace/trigger.ts

```diff
@@ -141,7 +141,7 @@ export function parseEventSource(event: any) {
     return eventTypes.lambdaUrl;
   }
   if (
-    eventType.isAPIGatewayEvent(event) ||
+    eventType.(event) ||
```

Comment:
Is this a bad copy paste ? it's a syntax error and i'm not sure what you tried to do ?